pyleadsheet/main.py

```diff
@@ -81,3 +82,6 @@ def main():
 
     elif args['generate']:
         return generate(args)
+
+if __name__ == '__main__':
+    main()
```

Comment:
I don't prefer to use this method of accessing from the command line.  Instead, I install the package into a virtual environment and the executable `pyleadsheet` becomes available, thus testing with the same point of entry that anyone actually installing the package from `pip` would experience.